src/components/calcite-picker-row/calcite-picker-row.tsx

```diff
@@ -0,0 +1,152 @@
+import { Component, Element, Event, EventEmitter, Host, Method, Prop, h } from "@stencil/core";
```

Comment:
was the PR to rename this file to `item` instead of `row` seperate?

Reply:
yes